examples/telehealth-tutorial-app/src/App.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import "./App.css";
+import { useRoomConnection } from "@whereby.com/browser-sdk";
+import IconButton from "./IconButton";
+import ChatInput from "./ChatInput";
+
+const ROOM_URL = "";
+
+function App() {
+    const [isCameraActive, setIsCameraActive] = React.useState(true);
+    const [isMicrophoneActive, setIsMicrophoneActive] = React.useState(true);
+    const [isLocalScreenshareActive, setIsLocalScreenshareActive] = React.useState(false);
+    const chatMessageBottomRef = React.useRef<HTMLDivElement>(null);
+
+    const roomConnection = useRoomConnection(ROOM_URL, {
```

Comment:
what do you think of adding a message if someone forgets to add a room url like I just did 😄 

![image](https://github.com/whereby/browser-sdk/assets/138459303/0c7a66a4-5505-496f-8971-17ec629adbab)

Reply:
ahh, good point. Yeah I can add it, I'm just not sure how much this should diverge from the tutorial 🤔 Since this is the "final" version of the tutorial (both the video and the written one), it might be confusing if have extra code here that is not present in the tutorials.. What do you think?

Reply:
mmm I think it should be fine, right? Even something as silly as 

`{roomUrl === "" && "Please set the room URL"}`

should do it?

Reply:
yeah, makes sense. I'll add it 👍

Reply:
This didn't actually work, as the whole component doesn't mount if there's no room url, so no text get rendered. I just added a comment about the room url in the code.